config/development.json

```diff
@@ -155,6 +155,7 @@
 		"onboarding/import-light": false,
 		"onboarding/import-redirect-to-themes": true,
 		"onboarding/interval-dropdown": true,
+		"onboarding/new-migration-instructions-step": true,
```

Comment:
Not a big deal, but I'm not sure that this is technically an onboarding flow. Could we put it under `migration-flow/` as [Serenity did previously](https://github.com/Automattic/wp-calypso/pull/90663/files#diff-811c6e3fd087924c069c3a6a662ecf053f21649dabd23857e6d6c431e0d7a17fL142)?

Reply:
Of course! Changed in https://github.com/Automattic/wp-calypso/pull/91820/commits/53409aa864db8bfbd044113850f9ce7604c783c9.